src/perl/vdotest/VDOTest.pm

```diff
@@ -253,15 +253,17 @@ sub _minimumMemory {
   # Fedora 38 measured 2023-07-24 needed 400M for pre-VDO operation.
   # Fedora 39 not measured; assume it needs 400M for pre-VDO operation.
   # Fedora 40 not measured; assume it needs 400M for pre-VDO operation.
+  # FEDORANEXT not measured; assume it needs 400M for pre-VDO operation.
   # RHEL8 measured 2023-07-24 needed 400M for pre-VDO operation.
   # RHEL9 measured 2023-07-24 needed 400M for pre-VDO operation. (VDO-5559)
   my %DISTRO_MEMORY_REQUIREMENTS
     = (
-       FEDORA39 => 400 * $MB,
-       FEDORA40 => 400 * $MB,
-       RAWHIDE  => 400 * $MB,
-       RHEL8    => 400 * $MB,
-       RHEL9    => 400 * $MB,
+       FEDORA39   => 400 * $MB,
+       FEDORA40   => 400 * $MB,
+       FEDORANEXT => 400 * $MB,
+       RAWHIDE    => 400 * $MB,
```

Comment:
I would slightly prefer that FEDORANEXT is listed after RAWHIDE because technically it will always be newer than RAWHIDE. But it's not a big deal.

Reply:
I can change it if you like but I put the list in alphabetical order.  Let me know,  Thanks!

Reply:
Either is plausible. You used alphabetical order elsewhere (e.g., in //eng/main) so it's fine here, too.